src/components/TeamPage.tsx

```diff
@@ -36,6 +39,14 @@ export const TeamPage = ({ teamId }: TeamPageProps) => {
                         children.map((child) => <GroupTreeViewNode key={child.id} group={child} visible />),
                     )}
                 </TreeView>
+                <PreviewContent>
+                    {nullable(groupQuery.data, (g) => (
+                        <>
+                            <TeamPeople groupId={g.id} isEditable={true} />
```

Comment:
Should it be always editable? 
For every user?

Reply:
Oh, no

Reply:
It`s hidden if user is`t an admin. Vadim added a component (```<Restricted visible={group.meta.isEditable}>``` ) that adjusts Visibility

#595